cornflow-server/cornflow/cli/service.py

```diff
@@ -83,11 +91,11 @@ def init_cornflow_service():
     os.environ["SIGNUP_ACTIVATED"] = str(signup_activated)
     user_access_all_objects = os.getenv("USER_ACCESS_ALL_OBJECTS", 0)
     os.environ["USER_ACCESS_ALL_OBJECTS"] = str(user_access_all_objects)
-    default_role = os.getenv("DEFAULT_ROLE", 2)
```

Comment:
Why this change? Why are you changing the default role on cornflow on general?

Reply:
In the constants file, role 2 is defined as PLANNER_ROLE. I am just using the constant instead of the number for better understanding

Reply:
The same with the rest of numbers

Reply:
`default_role = int(os.getenv("DEFAULT_ROLE", PLANNER_ROLE))`

Reply:
You have to covnert to int